src/components/VideoPlayer.vue

```diff
@@ -1,105 +1,106 @@
+            Current Time: <span v-text="currentTime"></span><br />
+            Loop Seconds: <span v-text="loopStartSeconds"></span> <span v-text="loopEndSeconds"></span><br />
+        </div>
+    </div>
 </template>
+
 <script>
 /* eslint-disable */
-    export default {
-        props: [this.comboId],
-        data: () => ({
-            videoId: this.comboId,
-            duration: 0,
-            loopStart: 0.16,
-            loopEnd: 1.05,
-            currentTime: null,
-            checkInterval: null
-        }),
-        computed: {
-            character() {
-                return this.$store.state.selectedCharacter;
-            },
-            loopStartSeconds() {
-                return this.loopStart * 60;  
-            },
-            loopEndSeconds() {
-                return this.loopEnd * 60;
-            }
+import YouTubePlayer from 'youtube-player';
+
+var YTPlayer;
+var player;
+var done = false;
+
+export default {
+    props: {
+        comboId: String
+    },
+    data: function() { return {
+        videoId: this.comboId,
+        duration: 0,
+        loopStart: 0.16,
+        loopEnd: 1.05,
+        currentTime: null,
+        checkInterval: null
+    }},
+    computed: {
+        character() {
+            return this.$store.state.selectedCharacter;
         },
-        watch: {
-            loopStart() {
-                player.seekTo(this.loopStartSeconds, true);
-            }
+        playerId() {
+            return "player" + this.videoId;
         },
-        methods: {
-          onPlayerReady(event) {
-              //event.target.playVideo();
-              this.duration = player.getDuration()/60;
-          },
-          onPlayerStateChange(event) {
-              if (event.data == YT.PlayerState.PLAYING && !done) {
-                  var self = this;
-                  console.log("starting loop");
-                  
-                  this.checkInterval = setInterval(function() {
-                      if (player.getCurrentTime)  
-                        self.currentTime = player.getCurrentTime();
-                    
-                      if (self.loopStart && self.loopEnd && self.currentTime > self.loopEndSeconds) {
-                          console.log("here");
-                          player.seekTo(self.loopStartSeconds, true);
-                      }
-                  }, 250)
-              } else if (event.data != YT.PlayerState.PLAYING) {
-                  console.log("ending loop");
-                  clearInterval(this.checkInterval);
-              }
-          },
-          playVideo() {
-                player.seekTo(this.loopStartSeconds, true);
-                player.playVideo();
-          },
-          stopVideo() {
-              player.stopVideo();
-          }
+        loopStartSeconds() {
+            return this.loopStart * 60;  
         },
-        mounted() {
-            function onYouTubeIframeAPIReady() {
```

Comment:
This is a method built into the YouTubeIframeAPI and won't work here. This why I had to switch to a NPM package.